src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -98,6 +100,59 @@ public function register_widget( $widgets ) {
 		return $widgets;
 	}
 
+	/**
+	 * Enable widget copy paste for the Legacy Widgets that we are registering.
+	 * 
+	 * @since TBD
+	 * 
+	 * @param mixed           $result  The result of the rest request.
+	 * @param WP_REST_Server  $server  The REST server.
+	 * @param WP_REST_Request $request The REST request.
+	 * 
+	 * @return mixed The result of the rest request.
+	 */
+	public function enable_widget_copy_paste( $result, $server, $request ) {
+		// Bail if result is already set.
+		if ( null !== $result ) {
+			return $result;
+		}
+	
+		// Get the route being requested.
+		$route = $request->get_route();
+		
+		// Check if this matches our target endpoint.
+		if ( ! preg_match( '#^/wp/v2/widget-types/([a-zA-Z0-9_-]+)/encode$#', $route, $matches ) ) {
+			return $result;
+		}
+	
+		// Get the widget type ID from the route.
+		$widget_type_id = $matches[1];
+	
+		global $wp_widget_factory;
+
+		$widget_object = $wp_widget_factory->get_widget_object( $widget_type_id );
```

Comment:
Move this below the next two conditionals as it is slightly more expensive and not required for either conditional conditional

Reply:
Oh. Actually, I don't think it is being used at all